pkg/common/entrypoint/entrypoint_windows.go

```diff
@@ -20,7 +25,10 @@ type systemCall struct {
 }
 
 func (s *systemCall) IsWindowsService() (bool, error) {
-	return svc.IsWindowsService()
+	// We are using a custom function because the svc.IsWindowsService() one still has an open issue in which it states
+	// that it is not working properly in Windows containers: https://github.com/golang/go/issues/56335. Soon as we have
```

Comment:
Is it worth submitting a PR.? That issue has a help wanted tag and it seems like you have a fix below.

Reply:
A PR is already opened for this, in fact, the custom function with the workaround is based on the opened solution proposal: https://github.com/golang/sys/pull/141